packages/amplify_auth_cognito/example/pubspec.yaml

```diff
@@ -28,9 +28,17 @@ dev_dependencies:
   flutter_test:
     sdk: flutter
   test: any
+  flutter_driver:
+    sdk: flutter
+  integration_test:
+    sdk: flutter
 # For information on the generic Dart part of this file, see the
 # following page: https://dart.dev/tools/pub/pubspec
 
+# Needed to avoid conflict for integration tests until flutter driver has null safe version of crypto https://github.com/flutter/flutter/issues/77282
```

Comment:
This is not needed anymore with flutter 2.2.0 but it probably makes sense to leave this here IMO until that's more of a standard dev practice to use.